ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/deprovision/cucushift-installer-rehearse-aws-ipi-disconnected-private-deprovision-chain.yaml

```diff
@@ -7,6 +7,5 @@ chain:
   - ref: aws-deprovision-security-group
   - ref: aws-deprovision-stacks
   - ref: aws-deprovision-s3buckets
-  - ref: aws-deprovision-users-and-policies
```

Comment:
This step is used to deprovision the iam users created by `aws-provision-cco-manual-users-static`